### PR TITLE
Define state default in the reducers

### PR DESCRIPTION
**Pull request by nchevobbe**
This is for https://github.com/devtools-html/gecko-dev/issues/136 
@linclark I don't know if I understood this well though

### CODE REVIEW

File: devtools/client/webconsole/new-console-output/reducers/filters.js [hunk 1]
```diff
@@ -23,4 +31,5 @@ function filters(state = new Filters(), action) {
   return state;
 }
 
+exports.Filters = Filters;
 exports.filters = filters;
```
Comment (nchevobbe) [inline devtools/client/webconsole/new-console-output/reducers/filters.js]: It feels weird to have the same name with different cases for different purposes

Comment (linclark) [inline devtools/client/webconsole/new-console-output/reducers/filters.js]: There's not a huge amount of best practice around using Immutable and Redux together, so this is just my opinion, but maybe having a convention where we append "State" (e.g. `FilterState`) would make sense.

**Comment by linclark** [comment]
This looks right to me, so once you change the export name this should be good to go.

**Comment by nchevobbe** [comment]
@linclark , I made the changes on the Record's names

**Comment by linclark** [comment]
Awesome, thank you!

**Pull request merged**